### PR TITLE
Rename automobile light and wheel

### PR DESCRIPTION
**Pull request by DavidMansolino**
Linked to omichel/webots#6395

### CODE REVIEW

**Comment by DavidMansolino** [comment]
👍

**Comment by stefaniapedrazzi** [comment]
:eyes:

File: automobile/vehiclewheel.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 
-The `AutomobileWheel` allows the user to easily create automobile wheels. It is
+The `VehicleWheel` allows the user to easily create vehicle wheels. It is
 designed to be generic and customizable to cover a wide range of wheel
-configurations. The base node of the `AutomobileWheel` PROTO is a `Slot` of type
-`automobile wheel` so that it can only be inserted as a wheel of an automobile
-PROTO featuring the corresponding `automobileWheel` slots.
+configurations. The base node of the `VehicleWheel` PROTO is a `Slot` of type
+`vehicle wheel` so that it can only be inserted as a wheel of a vehicle
+PROTO featuring the corresponding `VehicleWheel` slots.
```
Comment (stefaniapedrazzi) [inline automobile/vehiclewheel.md]: Before this was starting with a lower case letter.
Is this change made on purpose?

Comment (DavidMansolino) [reply automobile/vehiclewheel.md]: No, it is a search and replace issue.

**Pull request merged**